variables.tf

```diff
@@ -18,6 +18,12 @@ variable "server_name" {
   type        = string
 }
 
+variable "storage_tier" {
+  description = "Specifies the name of the Storage Tier. Changing this forces a new resource to be created."
+  type        = string
+  default     = "P15"
```

Comment:
can we make the storage_tier variable default = "p10". The default value for the storage_mb = 102400 which is near to the p10. 

Here is the like that shows the relation between storage_mb & storage_tier


https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/postgresql_flexible_server#storage_tier-defaults-based-on-storage_mb